app/views/facility_notifications/index.html.haml

```diff
@@ -10,7 +10,22 @@
   = render "shared/transactions/top", tab: "notifications"
 
 - if @order_details.any?
-  = render "shared/transactions/table", order_details: @order_details
+  #table_billing
+    = form_tag url_for(controller: params[:controller], action: :send_notifications), id: "journals_create_form", name: "journals_create_form", method: :post, class: "form-horizontal" do
+      .row
```

Comment:
It doesn't look like this view is using `:action_instructions`. Can we get rid of this here?